src/views/name.njk

```diff
@@ -51,6 +53,24 @@
       nameField.value = nameField.value.trim();
     });
   });
+
+  const content = document.getElementById('content');
+  const whichType = sessionStorage.getItem("which-type");
+  const whichTypeDb = content.dataset.type;
```

Comment:
Just wondering where the `content.dataset.type` value comes from? `content` is defined on line 57 but `dataset` and `type`?

Reply:
it is hard to get the data passed to the view in the javascript (<script>) because it is templating, only usable in the ‘html’ part


what I have done is add a dataset to the div used to render the content (line 5)
this one has the value stored in db if it exists

Reply:
Understand now.